cluster-up/scale.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/bash -x
+
+# Add Rhel 7.7 Worker
+
+CUSTOM_IMAGE=${CUSTOM_IMAGE:-"0"}
+PROVISION_MODE=${PROVISION_MODE:-"0"}
+REPO_FILE=${REPO_FILE:-"0"}
+
+BASE_IMAGE="rhel-server-7.7-x86_64-kvm.qcow2"
+export KUBECONFIG=/root/install/auth/kubeconfig
+
+SELF_PID=$(echo $$)
+ps -ef | grep scale.sh | grep -v grep | awk '{print $2}' | grep -v $SELF_PID | xargs kill
+pkill dnf
+pkill ansible-playbook
+
+dnf install -y libguestfs-tools
+if [ ! -f /root/.ssh/id_rsa.pub ]
+then
+   echo "id_rsa tuple must exists, Generating"
+   ssh-keygen -t rsa -f /root/.ssh/id_rsa -N ''
+fi
+
+dnf install -y https://repo.fedora.md/centos/7/virt/x86_64/ovirt-4.2/common/ansible-2.8.2-1.el7.noarch.rpm
+dnf install -y openshift-ansible openshift-clients jq
+
+NETLIST=$(virsh net-list | grep -v def | grep active | awk '{print $1}')
+
+cd /tmp
+if [ $CUSTOM_IMAGE = "0" ] || [ $PROVISION_MODE = "1" ]; then
+   qemu-img create -f qcow2 el77-worker-1.img 20G
+   virt-resize --expand /dev/sda1 $BASE_IMAGE el77-worker-1.img
+else
+   cp $CUSTOM_IMAGE el77-worker-1.img
+fi
+
+virt-customize -a el77-worker-1.img --root-password password:123456 --ssh-inject root:file:/root/.ssh/id_rsa.pub --selinux-relabel --timezone Europe/Berlin --hostname $NETLIST-worker-1 --uninstall cloud-init,kexec-tools,postfix
+virt-install --name $NETLIST-worker-1 --description "$NETLIST-worker-1" --os-type=linux --os-variant=rhel7.6 --vcpus=4 --ram 7168 --rng /dev/urandom --import --disk el77-worker-1.img --network network=$NETLIST --check all=off --connect qemu:///system --graphics none -m 52:54:00:72:5c:30 --noautoconsole --cpu host-passthrough
+
+function getip {
+  IP=$(virsh domifaddr $NETLIST-worker-1  | awk '{ print $4 }' | tail -2 | head -1 | awk -F/ '{print $1}')
+}
+
+function run_ssh {
```

Comment:
You can use HEREDOCS with ssh to run at embeeded script 
https://linuxize.com/post/bash-heredoc/#using-heredoc-with-ssh